libs/jmeter/jmeter_integration.py

```diff
@@ -94,23 +94,25 @@ def run_jmx(self, jmx_file: str, threads:int=25, rampup:int=1, loop:int=1, test_
     def run_verify_jmx(
         self,
         jmx_file: str,
+        expect_error_count = 0,
```

Comment:
Can u write a separate function for checking negative test scenario.
It should check for expected HTTP error and not just count.